Dustin-Guillen.txt

```diff
@@ -0,0 +1,138 @@
+	 A <section> is semantic and more specific and easier understood by screen readers, while the <div> is more general.
+
+    17. What is CSS used for? 
+	 Cascading Style Sheets are used to style HTML.
+
+    18. How do we select an element? Example - every h2 on the page
+	 You can select an element is CSS by writing its tag, writing its class with a dot before it, or writing it's id with a hash before it.
+	 Example:
+	 h2
+	 .h2
+	 #h2
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one.
+	 The differnece between a class and an id is that classes may be used on multiple elements as many times as needed per page, while id's are used to style ONE specific element.
+	 A class can be used for multiple differnt images in multiple differnt sections. 
+	 An id can be used to style one specific element in a section with multiples of the same element.
+
+    20. How do we select classes in CSS?
+	 You can select a class in CSS by writing the class name with a dot before it. (Example) .class
+
+    21. How do we select a p element with a single class of �human��?
+	 You would select the class of human by writing the class name with a dot before it. (Example) .human
+
+    22. What is a parent child selector? When would this be useful?
+	 The parent  child selector selects all elements that are a direct child of the specific element. 
+	 It would be useful when you want a more specific seletor without having to us id.
+ 
+    23. How do you select all links within a div with the class of sidebar?
+	 You write the class name sidebar with a dot in front of it, followed by the element tag a with nothing in front of it.
+	 
+    24. What is a pseudo selector?
+	 A pseudo selector is a class selector with a higher weight.
+	 It s written by writing the class selector with a colon and command following it with no spaces.
+		
+    25. What do we use the change the spacing between lines?
+	 The spacing between line can be changed using the CSS property of line height.
+	 
+
+    26. What do we use to change the spacing between letters?
+	 The spacing between letters can be changed using the CSS property letter-spacing.
+
+    27. What do we use to to change everything to CAPITALS? lowercase? Capitalize?
+	 All of the commands can be done with the CSS property text-transform.
+	 
+    28. How do I add a 1px border around my div that is dotted and black?
+	 The border can be created using the CSS property border, giving the property a number, design style and color.
+
+    29. How do I select everything on the page? 
+	 Every element on the page can be selected using the universal selector *.
+
+    30. How do I write a comment in CSS?
+	 Multiple lines comments can be made by typing /* as the open tag and */ as the close tag.
+
+    31. How do I find out what file I am in, when I am using the command line? 
+	 The file you are in appears in the line where you commands are written.
+
```

Comment:
there is a specific command you can use to `print` the file directory you are in. Do you remember what it is?